lib/fog/google/requests/storage_json/put_object_url.rb

```diff
@@ -0,0 +1,42 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # Get an expiring object url from Google Storage for putting an object
+        #
+        # ==== Parameters
+        # * bucket_name<~String> - Name of bucket containing object
+        # * object_name<~String> - Name of object to get expiring url for
+        # * expires<~Time> - An expiry time for this url
+        #
+        # ==== Returns
+        # * response<~Excon::Response>:
+        #   * body<~String> - url for object
+        #
+        def put_object_url(bucket_name, object_name, expires, headers = {})
```

Comment:
Unused method argument - `bucket_name`. If it's necessary, use `_` or `_bucket_name` as an argument name to indicate that it won't be used. You can also write as `put_object_url(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `object_name`. If it's necessary, use `_` or `_object_name` as an argument name to indicate that it won't be used. You can also write as `put_object_url(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `expires`. If it's necessary, use `_` or `_expires` as an argument name to indicate that it won't be used. You can also write as `put_object_url(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `headers`. If it's necessary, use `_` or `_headers` as an argument name to indicate that it won't be used. You can also write as `put_object_url(*)` if you want the method to accept any arguments but don't care about them.